Formula/inadyn.rb

```diff
@@ -0,0 +1,34 @@
+class Inadyn < Formula
+  desc "Dynamic DNS client with IPv4, IPv6, and SSL/TLS support"
+  homepage "http://troglobit.com/projects/inadyn/"
+  url "https://github.com/troglobit/inadyn/releases/download/v2.5/inadyn-2.5.tar.gz"
+  sha256 "f7faf0be4f0b4bfa1acc811189a9ed0a58bc367e48ea31c283920a2ef27cdc40"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "cmake"    => :build
+  depends_on "libtool"  => :build
+  depends_on "confuse"
+  depends_on "gnutls"
+  depends_on "pkg-config"
+
+  patch do
```

Comment:
```suggestion
  # Fix Sierra build, remove in next version
  patch do
```

Reply:
Everything you're asking for is reasonable, but can we avoid having a dozen separate requests for one-line changes?  Why not give them all to me at once?